src/resolveServer.js

```diff
@@ -55,6 +70,10 @@ module.exports = async function resolveServer(argv) {
     server.username = unsplat(argv.username);
     server.password = "";
   }
+  if (argv.passwordFile) {
+    delete server.token;
+    server.password = await readFile(argv.passwordFile, "utf-8");
```

Comment:
We should output a meaningful error message if this fails, don't you think?

Reply:
Also, do we handle line-endings? Plenty of editors add trailing newlines (or CRLF on Windows). We should probably trim those if we don't do that already. Probably makes sense to do this even if the password was prompted.